File: 02_Estructuras de Control/02_Condicionales/01_Simples/Python/ejemplo2.py
# Programa para identificar si un número es par

print("===== Verificación si un número es par =====\n")

# Entrada de datos
numero = int(input("Ingrese un número: "))

# Proceso + Salida de datos
if numero % 2 == 0:
    print("\033[92mEl número es par\033[0m")  # Verde

input("\nPresione Enter para salir...")

File: 02_Estructuras de Control/02_Condicionales/01_Simples/CSharp/ejemplo2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejemplo2Simple
{
    internal class Program
    {
        static void Main(string[] args)
        {
            /* Programa para identificar si un número es par
             */

            Console.ForegroundColor = ConsoleColor.Cyan;
            Console.WriteLine("===== Verificación si un número es par =====");
            Console.ResetColor();
            Console.WriteLine();

            // Entrada de datos

            Console.Write("Ingrese un número: ");
            int numero = int.Parse(Console.ReadLine());

            // Proceso + Salida de datos

            if (numero % 2 == 0)
            {
                Console.ForegroundColor = ConsoleColor.Green;
                Console.WriteLine("El número es par");
                Console.ResetColor();
            }

            Console.WriteLine("\nPresione cualquier tecla para salir...");
            Console.ReadKey();
        }
    }
}